Library/Models/LocalUser.cs

```diff
@@ -0,0 +1,79 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using Newtonsoft.Json;
+
+namespace Nfield.Models
+{
+    /// <summary>
+    /// Model representing the settable values in on a local user.
+    /// </summary>
+    public class LocalUserValues
+    {
+        /// <summary>
+        /// Name of user
+        /// </summary>
+        [JsonProperty]
```

Comment:
Do we also need these on all the public properties?

Reply:
I guess we probably don't? I will try remove them and see what happens.